### PR TITLE
improvement: use tilds ~ for dependencies version.

### PR DESCRIPTION
**Pull request by tomquinonero**
In semver, the first number stands for major change, the second for minor, the third for patch.
Caret "^" means only update patches, tild "~" means update also minor version.
Minor version does not break anything so let's use minor version updates.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,14 +31,19 @@
...
+		},
+		{
+			"email": "tquinonero@gmail.com",
+			"name": "Tom Quinonero",
+			"url": "https://github.com/tomquinonero"
 		}
 	],
 	"dependencies": {
-		"@3yourmind/yoco": "^1.1.0",
-		"color": "^3.1.0",
```
Comment (mes3yd) [inline package.json]: (1) I don't know how stable these other packages are, but IMHO the ones under our control should be ~. cc @amrdraz

File: package.json [hunk 2]
```diff
@@ -31,14 +31,19 @@
...
-		"dayjs": "^1.8.8",
-		"vue-clickaway": "^2.2.2",
-		"vue-popperjs": "^2.0.3"
+		"@3yourmind/yoco": "~1.1.0",
+		"color": "~3.1.0",
+		"dayjs": "~1.8.8",
+		"vue-clickaway": "~2.2.2",
```
Comment (amrdraz) [inline package.json]:
```suggestion
		"vue-clickaway": "^2.2.2",
```

File: package.json [hunk 3]
```diff
@@ -31,14 +31,19 @@
...
+		"vue-popperjs": "~2.0.3"
```
Comment (amrdraz) [inline package.json]:
```suggestion
		"vue-popperjs": "^2.0.3"
```

File: package.json (hunk 1 above)
Comment (amrdraz) [reply package.json]: I'm with @mes3yd I don't particularly trust, all packages, I've added a suggestion to change back the ones think may not respect minor version

**Pull request merged**